blank-canvas-blocks/sass/blocks/_group.scss

```diff
@@ -1,6 +0,0 @@
-// NOTE: Gutenberg sets a padding value by default for groups with a background.
```

Comment:
Removing this causes a difference in the editor and frontend for me, I just want to verify we're aware of that: 

Editor | Front-end
------ | ------
<img width="682" alt="Screen Shot 2021-03-18 at 5 55 51 PM" src="https://user-images.githubusercontent.com/5375500/111702800-0d524000-87fa-11eb-9a28-ae94bb3343ef.png"> | <img width="679" alt="Screen Shot 2021-03-18 at 5 55 46 PM" src="https://user-images.githubusercontent.com/5375500/111702812-117e5d80-87fa-11eb-805f-b6371c322c91.png">

Reply:
I understand that to be a reported Gutenberg issue where those styles are loaded when they shouldn't be.  I also thought that it was understood to be addressed though so perhaps it needs to be reopened.  I don't believe we should be attempting to overcome that difference here.

Reply:
Yes, this is the editor loading theme.css even though we removed the support for it on functions.php :(